apiary.apib

```diff
@@ -1708,6 +1708,17 @@ Get a list of all phases a deal can go through, sorted by their order in the flo
             + (object)
                 + id: `21efc56e-1ba8-469d-926a-e89502591b47` (string)
                 + name: `New` (string)
+                + deal_status: `new` (enum[string])
```

Comment:
why not just status?

Reply:
Can we replace this with a filter `excludeLost` that is only available for us? Feels like that solves the same problem and will be easier to keep hidden.

Reply:
So what if we then need to do something special for the won phase (which might be the case)? This doesn't seem like a good solution to me.

Reply:
It feels weird to limit the filter to only `Lost`, what if we want to hide the `Won` phase or another type of phase ?

Reply:
> It feels weird to limit the filter to only Lost, what if we want to hide the Won phase or another type of phase ?

yagni...

Although designs are changing towards a situation where we might want be able to identify the won phase... so we might have to swing back to the original idea... more info with @PieterReel